ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -49,8 +50,18 @@ define('setup_manage_taxonomies', ['reactaddons', 'lodash', 'jquery'],
         </li>
       </ul>;
     },
+    onEnterPress: function(e) {
+      var code = (e.keyCode ? e.keyCode : e.which);
+      if (code === 13) { //Enter keycode
+        this.handleAddTermClick();
+        return true;
```

Comment:
React does not support returning boolean values in event handlers, you need to use `event.preventDefault()` instead. I just learned about this yesterday, see [first warning box](https://facebook.github.io/react/docs/events.html)